src/threading.ts

```diff
@@ -247,11 +270,16 @@ export class BytecodeStackFrame implements IStackFrame {
 
     // Run until we get the signal to return to the thread loop.
     while (!this.returnToThreadLoop) {
-      var op = code.readUInt8(this.pc);
-      if (!RELEASE && logging.log_level === logging.VTRACE) {
-        vtrace(`  ${this.pc} ${annotateOpcode(op, this, code, this.pc)}`);
+      // var op = code.readUInt8(this.pc);
+      var op = method.getOp(this.pc, code);
+      if (typeof op === 'function') {
+        op(this, thread, jitUtil);
```

Comment:
Might be nice to have a VTRACE output that notes the call of a jitted function. IIRC, code behind RELEASE guards is stripped out for release builds, so it shouldn't cause any speed issues.